stackstorm_withitems/actions/with_items.py

```diff
@@ -0,0 +1,144 @@
+import copy
+import jinja2
+import re
+import os
+import six
+import time
+import urllib3
+
+from st2client.models import LiveAction
+from st2client.client import Client
+from st2client.commands import action as st2action
+from st2common.runners.base_action import Action
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
Sorry, I hit the wrong button there.

I would say we should make this configurable at most. While I suspect a lot of users will disable these warnings since most ST2 instances will have self-signed certs, I think we should have secure settings by default. We should include a section in the pack README about this though, since it will probably be a common question.

Reply:
I will take it out.